core/src/main/java/org/apache/accumulo/core/client/impl/TabletServerBatchWriter.java

```diff
@@ -831,11 +814,16 @@ public void send(TabletServerMutations<Mutation> tsm) throws AccumuloServerExcep
         try {
 
           long count = 0;
-          for (List<Mutation> list : mutationBatch.values()) {
-            count += list.size();
+
+          Set<Text> tableIds = new TreeSet<Text>();
+          for (Map.Entry<KeyExtent,List<Mutation>> entry : mutationBatch.entrySet()) {
+            count += entry.getValue().size();
+            tableIds.add(entry.getKey().getTableId());
           }
-          String msg = "sending " + String.format("%,d", count) + " mutations to " + String.format("%,d", mutationBatch.size()) + " tablets at " + location;
-          Thread.currentThread().setName(msg);
+          StringBuilder msg = new StringBuilder();
```

Comment:
fair enough, but is there any downside to using it here?  Especially if I am going to add the table ids myself instead of depending on the TreeSet implementation of toString()....